bpf/kmesh/bpf2go/bpf2go.go

```diff
@@ -18,10 +18,18 @@
 package bpf2go
 
 // go run github.com/cilium/ebpf/cmd/bpf2go --help
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang  --cflags $EXTRA_CFLAGS --cflags $EXTRA_CDEFINE KmeshCgroupSock ../ads/cgroup_sock.c -- -I../ads/include -I../../include -I../../../api/v2-c -DCGROUP_SOCK_MANAGE
```

Comment:
here we can discuss ， we don't need to redefine the KmeshCgroupSock to KmeshCgroupSockHi ，make it the default version？

Reply:
make sense

Reply:
correct